### ISSUE TITLE
Python User Guide sample code does not work with perspective 3

### ISSUE THREAD

**Issue by timkpaine** [finos/perspective#2757]
### Steps to Reproduce:
https://perspective.finos.org/docs/python/

```python
from datetime import date, datetime
import numpy as np
import pandas as pd
import perspective

data = pd.DataFrame({
    "int": np.arange(100),
    "float": [i * 1.5 for i in range(100)],
    "bool": [True for i in range(100)],
    "date": [date.today() for i in range(100)],
    "datetime": [datetime.now() for i in range(100)],
    "string": [str(i) for i in range(100)]
})

table = perspective.Table(data, index="float")
```

### Expected Result:
Code does not emit error

### Actual Result:
```raw
---> table = perspective.Table(data, index="float")

TypeError: No constructor defined
```

### Environment:
all

**Comment by texodus** [comment]
[You are already assigned as reviewer on the PR which fixes this](https://github.com/finos/perspective/pull/2754/files#diff-cc6d38c27ee2c73e90ab7dd57e9f688d443ca668a92c48ab385c739214670982R142)

**Comment by texodus** [comment]
Fixed in #2754, released in v3.1.0
